hazelcast/src/main/java/com/hazelcast/cache/impl/client/CacheAddPartitionLostListenerRequest.java

```diff
@@ -68,9 +67,8 @@ public void partitionLost(CachePartitionLostEvent event) {
                 new InternalCachePartitionLostListenerAdapter(listener);
         final EventFilter filter = new CachePartitionLostEventFilter();
         final ICacheService service = getService();
-        final EventRegistration registration = service.getNodeEngine().
-                getEventService().registerListener(AbstractCacheService.SERVICE_NAME,
-                name, filter, listenerAdapter);
+        final EventRegistration registration = service.getNodeEngine().getEventService()
+                .registerListener(ICacheService.SERVICE_NAME, name, filter, listenerAdapter);
         final String registrationId = registration.getId();
         endpoint.setListenerRegistration(CacheService.SERVICE_NAME, name, registrationId);
```

Comment:
Good catch. I have updated most of them as `ICacheService.SERVICE_NAME` but I had missed a few of them :)